app/deliveryoptions/utils.py

```diff
@@ -0,0 +1,727 @@
+        st += "<li>" + s + "</li>"
+    return st
+
+
+# Temporary markup
+def get_WebsiteUrls(record: dict, surrogate: List) -> str:
+    st = " "
+    for s in surrogate[1:]:
+        st += "<li>" + s + "</li>"
+    return st
+
+
+# Temporary markup
+def get_WebsiteUrl(record: dict, surrogate: List) -> str:
+    return surrogate[0]
+
+
+def get_website_url_text(record: dict, surrogate: List) -> str:
+    pattern = r">(.*?)<"
+
+    if len(surrogate):
+        match = re.search(pattern, surrogate[0])
+        return match.group(1) if match else ""
+    else:
+        return " "
+
+
+def get_your_order_link(record: dict, surrogate: List) -> str:
+    return "(Unknown order link)"
+
+
+# This dict links the embedded tags with a helper function that returns the
+# correct inserted value, e.g. url's
+deliveryOptionsTags = {
+    "{AccessConditionText}": get_access_condition_text,
+    "{AddedToBasketText}": get_added_to_basket_text,
+    "{AdvancedOrdersEmailAddress}": get_advanced_orders_email_address,
+    "{AdvanceOrderInformationUrl}": get_advance_order_information,
+    "{ArchiveLink}": get_archive_link,
+    "{ArchiveName}": get_archive_name,
+    "{BasketType}": get_basket_type,
+    "{BasketUrl}": get_basket_url,
+    "{BrowseUrl}": get_browse_url,
+    "{ContactFormUrlUnfit}": get_contact_form_url_unfit,
+    "{ContactFormUrlMould}": get_contact_form_url_mould,
+    "{ContactFormUrl}": get_contact_form_url,
+    "{DataProtectionActUrl}": get_data_protection_act_url,
+    "{DeptName}": get_dept_name,
+    "{DeptUrl}": get_dept_url,
+    "{DownloadFormat}": get_download_format,
+    "{DownloadText}": get_download_text,
+    "{DownloadUrl}": get_download_url,
+    "{FAType}": get_file_authority_type,
+    "{FoiUrl}": get_foi_url,
+    "{ImageLibraryUrl}": get_image_library_url,
+    "{ItemNumOfFilesAndSizeInMB}": get_item_num_of_files_and_size_in_MB,
+    "{KeepersGalleryUrl}": get_keepers_gallery_url,
+    "{KewBookingSystemUrl}": get_kew_booking_system_url,
+    "{MaxItems}": get_max_items,
+    "{OpenDateDesc}": get_open_date_desc,
+    "{OpeningTimesUrl}": get_opening_times_url,
+    "{OrderUrl}": get_order_url,
+    "{PaidSearchUrl}": get_paid_search_url,
+    "{Price}": get_price,
+    "{ReadersTicketUrl}": get_readers_ticket_url,
+    "{RecordCopyingUrl}": get_record_copying_url,
+    "{RecordInformationType}": get_record_information_type,
+    "{RecordOpeningDate}": get_record_opening_date,
+    "{RecordUrl}": get_record_url,
+    "{AllWebsiteUrls}": get_all_website_urls,
+    "{SubsWebsiteUrls}": get_subsequent_website_urls,
+    "{FirstWebsiteUrl}": get_first_website_url,
+    "{FirstWebsiteUrlFull}": get_first_website_url_full,
+    "{WebsiteUrlText}": get_website_url_text,
+    "{YourOrderLink}": get_your_order_link,
+}
+
+# Dictionary to serve as a cache for file contents, preventing redundant file reads
+file_cache = {}
+
+
+def read_delivery_options(file_path: str) -> List:
+    # Check if file content is already in the cache
```

Comment:
In this PR, one file where `filepath=app/deliveryoptions/delivery_options.json` 
Are we going to have more that one file to read?